### PR TITLE
printed out the correct error

### PR DESCRIPTION
**Pull request by cyberliem**

### CODE REVIEW

**Comment by cyberliem** [comment]
[Wrong Error printing at stat fetcher](https://trello.com/c/XWnQdhCJ/173-wrong-error-printing-at-stat-fetcher)

File: stat/fetcher.go [hunk 1]
```diff
@@ -872,7 +872,7 @@ func (self *Fetcher) FetchLogs(fromBlock uint64, toBlock uint64, timepoint uint6
 				}
 				SetcountryFields(&l)
 				if dbErr := self.CheckDupAndStoreTradeLog(l, timepoint); dbErr != nil {
-					log.Printf("LogFetcher - at block %d, storing trade log failed, stop at current block and wait till next ticker, err: %+v", l.BlockNo(), err)
+					log.Printf("LogFetcher - at block %d, storing trade log failed, stop at current block and wait till next ticker, err: %+v", l.BlockNo(), dbErr)
```
Comment (halink0803) [inline stat/fetcher.go]: It is checked for !=nil, you can print out %s with dbErr.Error().

File: stat/fetcher.go [hunk 2]
```diff
@@ -882,7 +882,7 @@ func (self *Fetcher) FetchLogs(fromBlock uint64, toBlock uint64, timepoint uint6
 					continue
 				}
 				if dbErr := self.CheckDupAndStoreCatLog(l, timepoint); dbErr != nil {
-					log.Printf("LogFetcher - at block %d, storing cat log failed, stop at current block and wait till next ticker, err: %+v", l.BlockNo(), err)
+					log.Printf("LogFetcher - at block %d, storing cat log failed, stop at current block and wait till next ticker, err: %+v", l.BlockNo(), dbErr)
```
Comment (halink0803) [inline stat/fetcher.go]: Same here.

File: stat/fetcher.go (hunk 1 above)
Comment (cyberliem) [reply stat/fetcher.go]: Currently all the error printing in our core is using err after assertion. minor change here is meaningless . If this is completely necessary, using err.Error() for all error printing, please create a new card to track this.

File: stat/fetcher.go (hunk 2 above)
Comment (cyberliem) [reply stat/fetcher.go]: same here

**Pull request merged**